ios/Classes/YotiFlutterPlugin.swift

```diff
@@ -55,7 +54,7 @@ public class YotiFlutterPlugin: NSObject, FlutterPlugin, YotiSDKDataSource, Yoti
       let sessionId: String = arguments?["sessionId"] as! String;
       let sessionToken: String = arguments?["sessionToken"] as! String;
 
-      self.startYoti(sessionId: sessionId ?? "", sessionToken: sessionToken ?? "", result: result)
+      self.startYoti(sessionId: sessionId, sessionToken: sessionToken, result: result)
```

Comment:
`sessionId` and `sessionToken` aren't nullable, so the right side would never be used. Fixed this to remove athewarning.